engine/api/repository/dao_vcs_project_repository_test.go

```diff
@@ -0,0 +1,55 @@
+package repository_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/ovh/cds/engine/api/database/gorpmapping"
+	"github.com/ovh/cds/engine/api/repository"
+	"github.com/ovh/cds/engine/api/test"
+	"github.com/ovh/cds/engine/api/test/assets"
+	"github.com/ovh/cds/engine/api/vcs"
+	"github.com/ovh/cds/sdk"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDecryption(t *testing.T) {
+	db, cache := test.SetupPG(t)
+
+	key1 := sdk.RandomString(10)
+	proj1 := assets.InsertTestProject(t, db, cache, key1, key1)
+
+	vcsProject := &sdk.VCSProject{
+		Name:        "the-name",
+		Type:        "github",
+		Auth:        sdk.VCSAuthProject{Username: "the-username", Token: "the-token"},
```

Comment:
this remind me some horrific stories...
could you test if the rollover is working as expected ?